libkbfs/md_util.go

```diff
@@ -507,7 +507,9 @@ func decryptMDPrivateData(ctx context.Context, codec kbfscodec.Codec,
 		}
 	}
 
-	// Re-embed the block changes if it's needed.
+	// Re-embed the block changes if it's needed.  TODO: we don't need
+	// to do this in minimal mode, since there's no node cache
```

Comment:
The re-embeded blocks would be used to get the list of changes in an MD update, which in turn would be used to update the BlockPointers in the node cache.  If there's no node cache, there's no need to inspect the ops list, and no need to re-embed the changes.  This would apply for reads as well as writes; anything that can create nodes.